src/components/profile/UserProgress/index.tsx

```diff
@@ -31,7 +33,7 @@ export const UserProgress = ({
     currentRank === nextLevelRank ? `Level ${getLevel(points + 100)}` : nextLevelRank;
 
   return (
-    <div className={styles.progressSection}>
+    <Link href={config.leaderboardRoute} className={styles.progressSection}>
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/6968e0a4-8a62-4cb7-a7ac-3c57f2e56791)

I think it's a bit weird that it's only the card content that is a link. Maybe you should turn the cards on the home and profile pages into a link instead of the component